trust-store-trust-policy-specification.md

```diff
@@ -158,41 +147,44 @@ The trust policy is represented as JSON data structure as shown below:
+
+The following table shows the resultant behavior `enforced` (verification fails), or `logged` for each of the checks, based on signature verification level.
+
+|Signature Verification Level|Recommended Usage|Integrity|Authenticity|Trusted timestamp|Expiry|Revocation check|
+|----------------------------|-----------------|---------|------------|-----------------|------|----------------|
+|strict    |Use at development, build and deploy time|enforced|enforced|enforced|enforced|enforced|
+|permissive|Use at deploy time or runtime|enforced|enforced|logged|logged|logged|
+|audit     |Use when adopting signed images, without breaking existing workflows|enforced|logged|logged|logged|logged|
+|skip      |Use to exclude verification for unsigned images|skipped|skipped|skipped|skipped|skipped|
+
+**Integrity** : Guarantees that the artifact wasn't altered after it was signed, or the signature isn't corrupted. All signature verification levels always enforce integrity.
+
+**Authenticity** : Guarantees that the artifact was signed by an identity trusted by the verifier. Its definition does not include revocation, which is when a trusted identity is subsequently untrusted because of a compromise.
+
+**Trusted timestamp** : Guarantees that the signature was generated when the certificate was valid. It also allows a verifier to determine if a signature be treated as valid when a certificate is revoked, if the certificate was revoked after the signature was generated. In the absence of a trusted timestamp, signatures are considered invalid after certificate expires, and all signatures are considered revoked when a certificate is revoked. **NOTE**: `notation` RC1 will generate trusted timestamp using a TSA when signature is generated, but will not support verification of TSA countersignatures.
+
+**Expiry** : This is an optional feature that guarantees that artifact is within “best by use” date indicated in the signature. Notary v2 allows users to include an optional expiry time when they generate a signature. The expiry time is not set by default and requires explicit configuration by users at the time of signature generation. The artifact is considered expired after the expiry time has elapsed, users performing verification can either configure their trust policies to fail the verification or even accept the signature with expiry date in the past using policy. This is an advanced feature that allows implementing controls for user defined semantics like deprecation for older artifacts, or block older artifacts in a production environment. Users should only include an expiry time in the signed artifact after considering the behavior they expect for consumers of the artifact after it expires. Users can choose to consume an artifact even after the expiry time based on their specific needs.
```

Comment:
expiry time has elapsed is unclear.  Do you mean when the current time is greater than or equal to the expiry time or just greater?

Reply:
Updated wording to be precise -  current time is greater than or equal to expiry time.

```suggestion
**Expiry** : This is an optional feature that guarantees that artifact is within “best by use” date indicated in the signature. Notary v2 allows users to include an optional expiry time when they generate a signature. The expiry time is not set by default and requires explicit configuration by users at the time of signature generation. The artifact is considered expired when the current time is greater than or equal to expiry time, users performing verification can either configure their trust policies to fail the verification or even accept the artifact with expiry date in the past using policy. This is an advanced feature that allows implementing controls for user defined semantics like deprecation for older artifacts, or block older artifacts in a production environment. Users should only include an expiry time in the signed artifact after considering the behavior they expect for consumers of the artifact after it expires. Users can choose to consume an artifact even after the expiry time based on their specific needs.
```